docs/docs/integrations/chat/yandex.ipynb

```diff
@@ -0,0 +1,109 @@
+{
```

Comment:
Is it possible to add the `Yandex integration card` page? See the [example](https://python.langchain.com/docs/integrations/providers/wikipedia). In your case, it would be not the `Document Loader` and `Retriever` sections but the `LLM` and `Chat model` sections.
Thanks! This is a very good integration!